storybook/src/utils/Gap/Gap.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { HTMLAttributes, PropsWithChildren } from 'react'
+
+const gapLengthOptions = ['xs', 'sm', 'md', 'lg', 'xl'] as const
+type GapLength = (typeof gapLengthOptions)[number]
+
+export type GapProps = {
+  /** The amount of vertical space between items. */
+  length: GapLength
```

Comment:
Wait, why is this called `length`? And not `gap`?

Reply:
It’s the length (width or height) of the gap. Not the gap of the Gap :)

Reply:
It still feels a little weird imo. How about 'size'?

Reply:
Not strongly opposed, but [length](https://developer.mozilla.org/en-US/docs/Web/CSS/length) is a very common term in CSS, and correct here, right?

Reply:
Updated to `size`.